maturity-level-one/tests/Codit.IntegrationTest/OpenApiTest.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Xunit;
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.AspNetCore.Hosting;
+using System.Net.Http;
+using System.Net;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Newtonsoft.Json.Linq;
+
+namespace Codit.IntegrationTest
+{
+    public class OpenApiTest
+    {
+        private readonly HttpClient _httpClient;
+        public OpenApiTest()
+        {
+            if (_httpClient != null) return;
+            var srv = new TestServer(new WebHostBuilder()
+                .UseEnvironment("Development")
+                .UseStartup<Codit.LevelOne.Startup>());
+
+            _httpClient = srv.CreateClient();
+        }
+
+        [Theory]
+        [InlineData("GET")]
+        public async Task OpenApi_OperationId_Test(string httpMethod)
```

Comment:
Maybe we need to check if there are better approaches for this?

Reply:
yes we can... anyway this is a simple implementation.  Let's move on with this and create an issue for a better implementation.